data/mods/MindOverMatter/powers/photokinesis.json

```
@@ -0,0 +1,334 @@
+    "energy_source": "STAMINA",
+    "difficulty": 3,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "base_energy_cost": 3000,
+    "final_energy_cost": 700,
+    "energy_increment": -125,
+    "base_casting_time": 250,
+    "final_casting_time": 100,
+    "casting_time_increment": -4.5,
+    "learn_spells": { "photokinetic_invisibility": 5, "photokinetic_light_flash": 7 }
+  },
+  {
+    "id": "photokinetic_invisibility",
+    "type": "SPELL",
+    "name": "[Ψ]Veil of Light",
+    "description": "Refract light around yourself to become invisible.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "PHOTOKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX", "RANDOM_DURATION" ],
+    "effect": "attack",
+    "effect_str": "effect_photokin_invisibility",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_four", "hit_self": true } ],
+    "shape": "blast",
+    "min_duration": {
+      "math": [
+        "( (u_val('spell_level', 'spell: photokinetic_invisibility') * 2000) + 1500) * (scaling_factor(u_val('intelligence') ) )"
+      ]
+    },
+    "max_duration": {
+      "math": [
+        "( (u_val('spell_level', 'spell: photokinetic_invisibility') * 2000) + 4000) * (scaling_factor(u_val('intelligence') ) )"
+      ]
+    },
+    "energy_source": "STAMINA",
+    "difficulty": 4,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "base_energy_cost": 3000,
+    "final_energy_cost": 1200,
+    "energy_increment": -125,
+    "base_casting_time": 250,
+    "final_casting_time": 100,
+    "casting_time_increment": -12.5,
+    "learn_spells": { "photokinetic_light_flash": 5, "photokinetic_light_disintegrate": 8 }
+  },
+  {
+    "id": "photokinetic_light_flash",
+    "type": "SPELL",
+    "name": "[Ψ]Star Flash",
+    "description": "Create a destructive flash of light.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "ground", "hostile" ],
+    "spell_class": "PHOTOKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "CONCENTRATE", "NO_HANDS", "NO_LEGS", "RANDOM_DAMAGE" ],
+    "effect": "attack",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_six", "hit_self": true } ],
+    "shape": "line",
+    "min_damage": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_flash') * 2) + 15) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "max_damage": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_flash') * 3) + 35) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "damage_type": "psi_photokinetic_damage",
+    "min_range": {
+      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_flash') * 0.6) + 2) * (scaling_factor(u_val('intelligence') ) )" ]
+    },
+    "max_range": 50,
+    "min_aoe": { "math": [ "u_val('spell_level', 'spell: photokinetic_light_flash') * (scaling_factor(u_val('intelligence') ) )" ] },
```

```suggestion
    "min_aoe":  {
      "math": [ "( (u_val('spell_level', 'spell: photokinetic_light_flash') * 0.3) + 2) * (scaling_factor(u_val('intelligence') ) )" ]
    },
```
Above is just a suggestion, but scaling directly with power level leads to a power that rapidly becomes fatter than it is long based on the range scaling.

---

Oh damn, missed that, thanks